### PR TITLE
Databricks Data Graph setup

### PR DESCRIPTION
**Pull request by forstisabella**
### Proposed changes
Added a procedure for setting up Databricks to the Data Graph/Linked Cinematic Universe

### Merge timing
Before 12 PM PST on Thursday, June 20th.

### CODE REVIEW

File: src/unify/linked-profiles/setup-guides/databricks-setup.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+> Linked Events is in private beta, and Segment is actively working on this feature. Some functionality may change before it becomes generally available. 
+
+On this page, you'll learn how to connect your Databricks data warehouse to your Segment Data Graph. 
+
+> info ""
+> At this time, you can only use Databricks with Linked Audiences.
+
+## Set up Databricks credentials
+
+Sign into Databricks with admin permissions to create new resources and provide Data Graph with the necessary permissions.
```
Comment (pwseg) [inline src/unify/linked-profiles/setup-guides/databricks-setup.md:18]:
```suggestion
Sign in to Databricks with admin permissions to create new resources and provide Data Graph with the necessary permissions. 
```

File: src/unify/linked-profiles/setup-guides/databricks-setup.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+Segment assumes that you already have a workspace that includes the datasets you'd like to use for the Data Graph. Segment recommends setting up a new Service Principal user with only the permissions to access the required catalogs and schemas.
+
+### Step 1: Set up a Service Principal and SQL Warehouse 
+
+Segment recommends that you set up a new Service Principal. If you already have a Service Principal you'd like to use, grant it "Can use" permissions for your data warehouse and proceed to [Step 2: Create a catalog for Segment to store checkpoint tables](#step-2-create-a-catalog-for-segment-to-store-checkpoint-tables). 
+
+To verify that your Service Principal has "Can use" permission, see the [Confirm Service Principal permissions](#confirm-service-principal-permissions) documentation. 
+
+#### Create a new Service Principal User
+1. Log into the Databricks UI as an Admin.
```
Comment (pwseg) [inline src/unify/linked-profiles/setup-guides/databricks-setup.md:29]:
```suggestion
1. Log in to the Databricks UI as an Admin.
```

File: src/unify/linked-profiles/setup-guides/databricks-setup.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+8. To add the user to the workspace:
+  1. Navigate to Workspaces and select your Workspace. 
+  2. Select the “Permissions” tab and click **Add Permissions**. 
+  3. Add the newly created Service Principal user and click **Save**. 
+
+> success ""
+> If you already have a warehouse you'd like to use, you can [Add your Service Principal User to Warehouse User Lists](#add-your-service-principal-user-to-warehouse-user-lists). If you need to create a new warehouse, see the [Create a new warehouse](#create-a-new-warehouse) documentation. 
+
+#### Add your Service Principal User to Warehouse User Lists 
+1. Log into the Databricks UI as an Admin.
```
Comment (pwseg) [inline src/unify/linked-profiles/setup-guides/databricks-setup.md:45]:
```suggestion
1. Log in to the Databricks UI as an Admin.
```

File: src/unify/linked-profiles/setup-guides/databricks-setup.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+3. Verify that the Service Principal has "Can use" permission. 
+
+### Step 2: Create a catalog for Segment to store checkpoint tables
+
+> warning "Segment recommends creating an empty catalog for Data Graph"
+> If you plan to use an existing catalog with Reverse ETL, follow the instructions in the [Update user access for Segment Reverse ETL catalog](#update-user-access-for-segment-reverse-etl-catalog) section.
+ 
+Segment requires write access to a catalog to create a schema for internal bookkeeping, and to store checkpoint tables for the queries that are executed. 
+
+Segment recommends creating an empty catalog for this purpose by running the SQL below. This is also the catalog that you'll be required to specify when setting up your Databricks integration in the Segment app.
```
Comment (pwseg) [inline src/unify/linked-profiles/setup-guides/databricks-setup.md:66]:
```suggestion
Segment recommends creating an empty catalog for this purpose by running the SQL below. This is also the catalog that you'll be required to specify when setting up your Databricks integration in the Segment app. 
```

This isn't a huge deal, but can we change the instances of "below" to "following"?  Accessibility and so on.

File: src/unify/linked-profiles/setup-guides/databricks-setup.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+USE SCHEMA `${schema_1}`;
+GRANT SELECT ON TABLE `${table_1}` TO `${client_id}`;
+GRANT SELECT ON TABLE `${table_2}` TO `${client_id}`;
+...
+
+```
+
+### Step 5: Validate the permissions of your Service Principal user
+
+Sign into the [Databricks CLI with your Client ID secret](https://docs.databricks.com/en/dev-tools/cli/authentication.html#oauth-machine-to-machine-m2m-authentication){:target="_blank”} and run the following SQL to verify the Service Principal user has the correct permissions for a given table.
```
Comment (pwseg) [inline src/unify/linked-profiles/setup-guides/databricks-setup.md:121]:
```suggestion
Sign into the [Databricks CLI with your Client ID secret](https://docs.databricks.com/en/dev-tools/cli/authentication.html#oauth-machine-to-machine-m2m-authentication){:target="_blank”} and run the following SQL to verify the Service Principal user has the correct permissions for a given table. 
```

Re: "below"/"the following", this one is great.

File: src/unify/linked-profiles/setup-guides/databricks-setup.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+- **Catalog**: The catalog you designated in [Step 2: Create a catalog for Segment to store checkpoint tables](#step-2-create-a-catalog-for-segment-to-store-checkpoint-tables)
+- **Service principal client ID**: The client ID used to access to your Databricks warehouse
+- **OAuth secret**: The OAuth secret used to connect to your Databricks warehouse
+
+After identifying the following settings, continue setting up your Data Graph by following the instructions in [Connect your warehouse to the Data Graph](/docs/unify/linked-profiles/data-graph/#step-2-connect-your-warehouse-to-the-data-graph). 
+
+## Additional set up for warehouse permissions
+
+#### Create a new warehouse
+1. Log into your workspace as an Admin in the Databricks UI.
```
Comment (pwseg) [inline src/unify/linked-profiles/setup-guides/databricks-setup.md:148]:
```suggestion
1. Log in to your workspace as an Admin in the Databricks UI.
```

**Review by pwseg — APPROVED** [review]
Some super minor stuff, but this looks great.

File: src/unify/linked-profiles/setup-guides/databricks-setup.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+CREATE CATALOG IF NOT EXISTS `SEGMENT_LINKED_PROFILES_DB`;
+-- Copy the Client ID by clicking “Generate secret” for the Service Principal user
+GRANT USAGE ON CATALOG `SEGMENT_LINKED_PROFILES_DB` TO `${client_id}`;
+GRANT CREATE ON CATALOG `SEGMENT_LINKED_PROFILES_DB` TO `${client_id}`;
+GRANT SELECT ON CATALOG `SEGMENT_LINKED_PROFILES_DB` TO `${client_id}`;
+```
+
+### Step 3: Grant read-only access to the Profiles Sync catalog
+
+Run the SQL below to grant the Data Graph read-only access to the Profiles Sync catalog:
```
Comment (forstisabella) [inline src/unify/linked-profiles/setup-guides/databricks-setup.md:77]:
```suggestion
Run the following SQL to grant the Data Graph read-only access to the Profiles Sync catalog:
```

File: src/unify/linked-profiles/setup-guides/databricks-setup.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+
+```sql
+GRANT USAGE, SELECT, USE SCHEMA ON CATALOG `${profiles_sync_catalog}` TO `${client_id}`;
+```
+
+### Step 4: Grant read-only access to additional catalogs for the Data Graph
+Run the SQL below to grant your Service Principal user read-only access to any additional catalogs you want to use for the Data Graph:
```
Comment (forstisabella) [inline src/unify/linked-profiles/setup-guides/databricks-setup.md:84]:
```suggestion
Run the following SQL to grant your Service Principal user read-only access to any additional catalogs you want to use for the Data Graph: 
```

File: src/unify/linked-profiles/setup-guides/databricks-setup.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+
+```sql
+GRANT USAGE, SELECT, USE SCHEMA ON CATALOG `${profiles_sync_catalog}` TO `${client_id}`;
+```
+
+### Step 4: Grant read-only access to additional catalogs for the Data Graph
+Run the SQL below to grant your Service Principal user read-only access to any additional catalogs you want to use for the Data Graph: 
+
+```sql 
+-- Run the SQL below for each catalog you want to use for the Segment Data Graph
```
Comment (forstisabella) [inline src/unify/linked-profiles/setup-guides/databricks-setup.md:87]:
```suggestion
-- Run this command for each catalog you want to use for the Segment Data Graph
```

File: src/unify/linked-profiles/setup-guides/databricks-setup.md [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+> Linked Profiles (Data Graph, Linked Events, and Linked Audiences) is in public beta, and Segment is actively working on this feature. Some functionality may change before it becomes generally available.
+
+On this page, you'll learn how to connect your Databricks data warehouse to the Segment Data Graph. 
+
+> info ""
+> At this time, you can only use Databricks with Linked Audiences.
+
+## Set up Databricks credentials
+
+Sign into Databricks with admin permissions to create new resources and provide the Data Graph with the necessary permissions.
```
Comment (forstisabella) [inline src/unify/linked-profiles/setup-guides/databricks-setup.md:18]:
```suggestion
Sign in to Databricks with admin permissions to create new resources and provide the Data Graph with the necessary permissions. 
```

File: src/unify/linked-profiles/setup-guides/databricks-setup.md [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+USE SCHEMA `${schema_1}`;
+GRANT SELECT ON TABLE `${table_1}` TO `${client_id}`;
+GRANT SELECT ON TABLE `${table_2}` TO `${client_id}`;
+...
+
+```
+
+### Step 5: Validate the permissions of your Service Principal user
+
+Sign into the [Databricks CLI with your Client ID secret](https://docs.databricks.com/en/dev-tools/cli/authentication.html#oauth-machine-to-machine-m2m-authentication){:target="_blank”} and run the following SQL to verify the Service Principal user has the correct permissions for a given table.
```
Comment (forstisabella) [inline src/unify/linked-profiles/setup-guides/databricks-setup.md:120]:
```suggestion
Sign in to the [Databricks CLI with your Client ID secret](https://docs.databricks.com/en/dev-tools/cli/authentication.html#oauth-machine-to-machine-m2m-authentication){:target="_blank”} and run the following SQL to verify the Service Principal user has the correct permissions for a given table. 
```

**Pull request merged**